main/lsp/requests/hover.cc

```diff
@@ -48,20 +75,38 @@ LSPResult LSPLoop::handleTextDocumentHover(unique_ptr<core::GlobalState> gs, con
         }
 
         auto resp = move(queryResponses[0]);
+
+        optional<string> documentation = nullopt;
+        if (
```

Comment:
You probably don't want to `findDocumentation` for identifiers (local variables) or literals (e.g. 1, "a string", etc). Those don't generally have documentation associated with them.

Reply:
👍  0bcf94e2a